pkg/workos_errors/errors.go

```diff
@@ -0,0 +1,12 @@
+package workos_errors
+
+import (
+	"errors"
+	"github.com/workos-inc/workos-go/internal/workos"
+	"net/http"
+)
+
+func ErrIsBadRequest(err error) bool {
```

Comment:
Could we change this name to `IsBadRequest`?

Reply:
@maxdeviant done 👍